staging/src/k8s.io/client-go/util/flowcontrol/backoff.go

```diff
@@ -136,9 +167,26 @@ func (p *Backoff) DeleteEntry(id string) {
 	delete(p.perItemBackoff, id)
 }
 
+// StartBackoffGC used to start the Backoff garbage collection mechanism
+// Backoff.GC() will be executed each minute
+func StartBackoffGC(backoff *Backoff, stopCh <-chan struct{}) {
```

Comment:
Not sure if this belongs here, but I'll defer to @deads2k for the final judgment.